helm/ovn-kubernetes/values-no-ic.yaml

```diff
@@ -138,6 +138,8 @@ global:
   ofctrlWaitBeforeClear: ""
   # -- Separate log file for libovsdb client 
   libovsdbClientLogFile: ""
+  # -- Label for indicating that the node is managing its own network
```

Comment:
NIT: Do we also need this in [values-multi-node-zone.yaml](https://github.com/ovn-org/ovn-kubernetes/blob/master/helm/ovn-kubernetes/values-multi-node-zone.yaml) and [values-single-node-zone.yaml](https://github.com/ovn-org/ovn-kubernetes/blob/master/helm/ovn-kubernetes/values-single-node-zone.yaml) ?
https://github.com/ovn-org/ovn-kubernetes/tree/master/helm/ovn-kubernetes